tda/tp0.md

```diff
@@ -14,28 +14,47 @@ aparecen en el código fuente: `Swap()`,  `Maximo()`, `Comparar()`, `Seleccion()
 
 De todos los archivos que pueden encontrar en el sitio de descargas, sólo deben modificar `tp0.go`. 
-El resto de archivos del zip no los deben modificar. Sí pueden, no obstante, leer el archivo `tp0_test.go` para entender qué verificaciones realizaremos sobre su código. Además deben de tener el archivo `go.mod` que es el árchivo del módulo del tp. Los 3 archivos deben encontrarse en el mismo directorio para poder ejecutar correctamente. 
+El resto de archivos del zip no los deben modificar. Sí pueden, no obstante, leer el archivo `tp0_test.go` para entender qué verificaciones realizaremos sobre su código. 
+
+Para esta parte del tp0, deben crear un paquete que contenga tanto a `tp0.go` como a `tp0_test.go`. Dicho paquete ha de llamarse `ejercicios`. Luego, fuera de este paquete (en el nivel principal del módulo a implementar) debe haber un archivo que puede llamarse como gusten (por ejemplo `main.go`) que tenga el paquete `main` sí o sí, donde estará el programa a implementar. 
+Además deben de tener el archivo `go.mod` que es el árchivo del módulo del tp. Es decir, para funcionar correctamente deben tener una estructura de directorios como la siguiente: 
+
+```
+TP0 (o como gusten llamarle al directorio del tp)
+| -> main.go
+| -> go.mod
+| -> ejercicios
+     | -> tp0.go
+     | -> tp0_test.go
+```
 
 La compilación (estando ubicados en el directorio donde se encuentren los archivos) se realiza con el siguiente comando:
 
     go build tp0
 
 Las pruebas deben dar todas OK, y las pueden ejecutar con:
 
+    cd ejercicios
     go test tp0_test.go
 
 O simplemente:
-	
+
+	cd ejercicios
 	go test
 	
+
+Luego que todas las pruebas ejecuten correctamente, deben implementar un programa (en el archivo principal, `main.go` que mencionamos antes) que: 
+* Lea dos archivos fijos (se llaman si o si `archivo1.in` y `archivo2.in`). Cada archivo tendrá dentro escrito un arreglo en texto, donde cada componente está separado por comas. Es decir, un ejemplo podría ser que en `archivo1.in` hay escrito `1, 7, 12, 8, 2, 1`. Cada archivo tiene una única línea, que define un único arreglo. Deben leer cada uno de los archivos, y obtener los arreglos correspondientes, separando cada componente (para lo que pueden utilizar la función `split` del módulo `strings`) y convertir cada uno de los elementos a `int` (para lo que pueden utilizar la función `Atoi` del módulo `strconv`). Pueden asumir que no van a haber errores en las lecturas, ni letras donde deben haber números, etc... 
+* Una vez obtenidos los arreglos, deben imprimir **ordenado** el arreglo que sea **mayor** entre los dos. Para esto, utilizar las funciones `Comparar` y `Seleccion` previamente implementadas. Para imprimir, hacerlo en el mismo formato que en el que vienen en los archivos. Es decir, cada componente separados por una coma. Si el arreglo más grande fuera `2, 1, 7, 3`, habría que imprimir `1, 2, 3, 7`. 
```

Comment:
nit:
* Abra y lea de dos archivos de texto regulares (llamados `archivo1.in` y `archivo2.in`). Cada archivo tendrá únicamente una línea de texto que será la representación de un arreglo de números. Cada elemento del arreglo estará separado por comas. Por ejemplo, el contenido de `archivo1.in` podría ser `1,7,12,8,2,1`.
* Se debe leer cada uno de los archivos, y cargar en memoria los arreglos correspondientes, separando cada componente (para lo que pueden utilizar la función `split` del módulo `strings`) y convertir cada uno de los elementos a `int` (para lo que pueden utilizar la función `Atoi` del módulo `strconv`). Pueden asumir que no van a haber errores en las lecturas, ni letras donde deben haber números, etc...
* Una vez obtenidos los arreglos, se debe imprimir por consola el arreglo que sea **mayor** entre los dos,  **ordenado**. Para esto, utilizar las funciones `Comparar` y `Seleccion` previamente implementadas. Para imprimir, hacerlo en el mismo formato que en el que vienen en los archivos. Es decir, cada componente separados por una coma. Si el arreglo más grande fuera `2,1,7,3`, habría que imprimir `1,2,3,7`.

Reply:
idem jaja

Reply:
```suggestion
* Abra y lea de dos archivos de texto regulares (llamados `archivo1.in` y `archivo2.in`). Cada archivo tendrá únicamente una línea de texto que será la representación de un arreglo de números. Cada elemento del arreglo estará separado por comas. Por ejemplo, el contenido de `archivo1.in` podría ser `1, 7, 12, 8, 2, 1`.
* Se debe leer cada uno de los archivos, y cargar en memoria los arreglos correspondientes, separando cada componente (para lo que pueden utilizar la función `split` del módulo `strings`) y convertir cada uno de los elementos a `int` (para lo que pueden utilizar la función `Atoi` del módulo `strconv`). Pueden asumir que no van a haber errores en las lecturas, ni letras donde deben haber números, etc...
* Una vez obtenidos los arreglos, se debe imprimir por consola el arreglo que sea **mayor** entre los dos, **ordenado**. Para esto, utilizar las funciones `Comparar` y `Seleccion` previamente implementadas. Para imprimir, hacerlo en el mismo formato que en el que vienen en los archivos. Es decir, cada componente separados por una coma. Si el arreglo más grande fuera `2, 1, 7, 3`, habría que imprimir `1, 2, 3, 7`.
```